assets/scss/styles.scss

```diff
@@ -0,0 +1,4 @@
+---
+---
```

Comment:
Jekyll only convert sass files if the .scss starts with two lines of triple dashes, so this is required to generate `scss/styles.css`